src/moby/util.go

```diff
@@ -1,6 +1,15 @@
 package moby
 
+import (
+	"strings"
+)
+
 var (
 	// MobyDir is the location of the cache directory which should be set by the caller
 	MobyDir string
 )
+
+// we are largely handling Linux paths, so do not want to use filepath.Join()
+func filepathJoin(paths ...string) string {
+	return strings.Join(paths, "/")
```

Comment:
`filepath.Join()` also does a `filepath.Clean()`. This may not be relevant here...